systemd/gluster-block.logrotate

```diff
@@ -0,0 +1,18 @@
+# Rotate all the gluster-block logs
+/var/log/gluster-block/*.log {
```

Comment:
This should be configurable. As our log-path is dynamic too.

Reply:
Yeah, will fix it.

Reply:
@lxbsz JFYI, this is pending on this PR.